criteria-queries/detached-criteria-builder/associations.md

```diff
@@ -21,7 +21,7 @@ var users = c.like("name","lui%")
      .list();
 ```
 
-You can also use a hibernate property approach which aliases the association much how HQL approaches it by using the createAlias\("associationName","alias"\) method:
+You can also use a hibernate property approach which aliases the association much how HQL approaches it by using the `createAlias\("associationName","alias"\)` method:
```

Comment:
Can you please rename all the `createAlias()` to `joinTo()` since that will be the new syntax going forward

Reply:
Done!